packages/client-sdk-nodejs/src/index.ts

```diff
@@ -143,6 +143,8 @@ import {
   TopicSelector,
   AllCaches,
   AllTopics,
+  LeaderboardOrder,
+  ILeaderboard,
```

Comment:
thank you! do we need to do this in the web sdk too?

Reply:
yep did that and mistakenly committed my local `npm i` installation.. will update examples once released!